src/domain/single-project/gallery-slider.tsx

```diff
@@ -50,7 +51,7 @@ export const GallerySlider = memo(({ images, gap }: Props) => {
                 <MotionSlider gap={gap} displayGrabCursor={false}>
                     {images.map(({ image }, i) => (
                         <Element key={i}>
-                            <img src={image} alt="" />
+                            <Image src={image} alt="" height="100" width="100" />
```

Comment:
These sizes may not be proper. I bet we need to recheck it

Reply:
Ye for size we need to handle it differently, again this was temporary solutions to lunch the broken project without endless warnings & errors